packages/react-i18n/src/i18n.ts

```diff
@@ -314,6 +315,29 @@ export class I18n {
     return getCurrencySymbol(locale, {currency});
   }
 
+  formatName(firstName: string, lastName?: string, options?: {full?: boolean}) {
+    if (!firstName) {
+      return lastName;
```

Comment:
Should it be `return lastName || ''`?
If `lastName` is missing, your function will return `undefined`.